### DISCUSSION TITLE
Font: from .h to .bdf?

### DISCUSSION THREAD

**Discussion by XFer012** [olikraus/u8g2#2099] [Q&A] [answered]
Hello,
I don't think it's possible, but had to ask... is it possible to "go back" from .h array to .bdf font?

I have a font in U8G2 format (header file) and would like to edit it in Fony, but I do not have the original font (.bdf/.fon)...

Thanks

**Comment by olikraus** [comment, marked as answer]
Depends. Is this an original u8g2 font? If so, what is the font name? 
The font source is mentioned here: https://github.com/olikraus/u8g2/blob/3500b1056b33999b285387d196c78fe4ab707f79/tools/font/build/build.c#L233

By knowing the font source you could use that source to modify it in Fony.

**Comment by XFer012** [comment]
It was a font I created time ago but lost the original material: all I had was the .h
I ended up re-creating it from scratch in Fony.
Thanks anyway!
